src/systempolicy/systemPolicy.go

```diff
@@ -1007,14 +1007,17 @@ func updateSysPolicySelector(clusterName string, pod types.Pod, policies []types
 	return results
 }
 
-func updateSysPolicies() {
+// UpdateSysPolicies updates system policy
+func UpdateSysPolicies(wpfsPolicies []types.KnoxSystemPolicy) {
 
 	var locSysPolicies []types.KnoxSystemPolicy
 	var isPolicyExist bool
 
-	wpfsPolicies := populateKnoxSysPolicyFromWPFSDb("", "", "", "")
+	if len(wpfsPolicies) < 1 {
```

Comment:
@vishnusomank Why was this check introduced ? I think it should be **> 1**

Reply:
since the hardening policies are going to use the same function from `system`, I am sending an empty `wpfsPolicies` array from system policy and a populated `wpfsPolicies` from hardening policies. To distinguish both discovered and hardening policies this check was introduced